### DISCUSSION TITLE
Simpleton error

### DISCUSSION THREAD

**Discussion by L-HALLW** [apertium/organisation#39] [Q&A] [answered]
Trying to use the Simpleton UI on windows 10, however when i press run i get the output:

Error: Grammar revision is 13897, but this loader only knows up to revision 13892!
Error: Grammar revision is 13897, but this loader only knows up to revision 13892!
terminate called after throwing an instance of 'std::runtime_error'
  what():  locale::facet::_S_create_c_locale name not valid

how can i solve this problem?

**Comment by TinoDidriksen** [comment, marked as answer]
That's because the Windows builds are not up to date. For now, I recommend running Apertium in a WSL Ubuntu environment: https://wiki.apertium.org/wiki/Installation#Install_Apertium_Core_by_packaging.2Fvirtual_environment
